v0/destinations/delighted/util.js

```diff
@@ -0,0 +1,114 @@
+const axios = require("axios");
+const { CustomError, getValueFromMessage } = require("../../util");
+const { ENDPOINT } = require("./config");
+
+const isValidEmail = email => {
+  const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  return re.test(String(email).toLowerCase());
+};
+const isValidPhone = phone => {
+  const phoneformat = /^\+[1-9]\d{10,14}$/;
+  return phoneformat.test(String(phone));
+};
+
+const isValidUserIdOrError = (channel, userId) => {
+  if (channel === "email") {
```

Comment:
instead of nesting if blocks can we use the `&&` operator instead?

Reply:
if we do `&&` then even the cases where channel and userid is correct it will go to else part instead of return.